src/Controls/src/Core/Element.cs

```diff
@@ -9,7 +9,7 @@
 
 namespace Microsoft.Maui.Controls
 {
-	public abstract partial class Element : BindableObject, IElement, INameScope, IElementController
+	public abstract partial class Element : BindableObject, IElement, INameScope, IElementController, IVisualTreeHelper
```

Comment:
I wonder if this should be on IElement? The all the other frameworks get it automatically, like Comet?

Reply:
I agree, I've updated and put IVisualTreeHelper on IElement (https://github.com/dotnet/maui/pull/1845/commits/28811af9faca00ef1526a7f8a8c606d9ca7a6de5). But I'm also not sure if GetVisualChildren should just collapse onto IElement instead, and it should be IElement.GetVisualChildren(). What does everyone think?